ONSdigital/census-rh-ui
# Cr 1429 fix to start confirm address

# Motivation and Context
Fixes to start>confirm-address page

# What has changed
Fix to error display when no option is selected.
Reinstatement of missing trigger to display the error message on the welsh start>confirm-address when an English UAC is used, and new unit test to check it

No Cucumber updates are required

# How to test?
Enter a UAC. On the confirm address page click continue without selecting an option. Check the error boxes display correctly
Enter an 'E' region UAC on /cy/start/. Check a panel box is displayed with a message (in welsh) on the confirm-address page

# Links
https://collaborate2.ons.gov.uk/jira/browse/CR-1429

# Screenshots (if appropriate):
